resources/ts/Shared/Footer.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export default function Footer() {
+  return (
+    <footer>
+      <p>Copyright 2022 publiq</p>
```

Comment:
Make year dynamic ;) so you don't have to change it each year

Reply:
You're totally right! This PR is focussing on the frontend tooling. In the actual frontend development we can change these things.

Reply:
Ok 👍